### DISCUSSION TITLE
Is there a way to customize the code font/theme for google colab? how?

### DISCUSSION THREAD

**Discussion by bide-xiong** [mrdbourke/pytorch-deep-learning#955] [Q&A]
Hello,

I found the way the python code displays on my computer differs from what Daniel has in his course.  The code display I'm having is way less readable.

First one is Daniel's
<img width="368" alt="pic1" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/72774581/02dadcd3-18ed-4f92-96f0-5e463861f4e7">

This second one is mine
<img width="275" alt="pic2" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/72774581/a453eb8c-24fc-4af4-974e-5549723f8ed9">

How to customize the code font/theme for google colab?

thanks

**Comment by hemanthkotaprolu** [comment]
You can go to `Tools > Settings`. Then wherever you want the theme to change you can customize it.

**Reply by bide-xiong** [reply]
Thanks, Kotaprolu.

However, I only noticed three options in `Tools > Settings > Site > Theme`, light/dark/adaptive, they don't help. It this what you mean?

Or should I change `Tools > Settings > Editor`?

**Comment by hemanthkotaprolu** [comment]
I guess other than light and dark there's no other option available. But I came across a Chrome extension for the google colab themes. Maybe try them once you may like them.

[Chrome extension](https://github.com/DannyCol/Colab-Themes)
